projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/progress-button.component.migration.ts

```diff
@@ -0,0 +1,17 @@
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+import {
+  CLIK_EVENT,
+  PROGRESS_BUTTON_COMPONENT,
+  SPARTACUS_STOREFRONTLIB,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+
+export const PROGRESS_BUTTON_COMPONENT_MIGRATION: MethodPropertyDeprecation[] =
```

Comment:
forgot to import this to `projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/methods-and-properties-deprecations.ts`

Reply:
let's add this relative path to file too as comment